### DISCUSSION TITLE
how to export vim-like marks

### DISCUSSION THREAD

**Discussion by zxyqq** [brookhong/Surfingkeys#1844] [Q&A] [answered]
how to export vim-like marks for later use

**Comment by zxyqq** [comment]
or provide to way for user to set the vim marks in config file

**Comment by zxyqq** [comment]
can anyone tell me the place where does the vim-like marks store?

**Comment by b0o** [comment, marked as answer]
`yj` to copy the current settings to your clipboard, then you can paste in into a text editor and search for "marks". You can also use `;pj` to restore settings from clipboard.

**Reply by zxyqq** [reply]
Thanks a lot!
